src/pages/Home/index.tsx

```diff
@@ -1,7 +1,7 @@
 import { Container } from 'react-bootstrap';
 import { useEffect, useState } from 'react';
 import { OrderProps, BobaProps } from 'types/common/main';
-import { bobaList, toppingsList, milkList } from 'assets/sampleBobaAPI';
+import { toppingsList, milkList } from 'assets/sampleBobaAPI';
```

Comment:
Since there's no JSON data, netlify hosting is stuck at loading

Reply:
updated 57d15d43a8edae133c80140c5cf2abb035c6aaf0